hack/install-database.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: databases
+---
```

Comment:
Why is this YAML embedded in a bash script instead of being in a separate file?

Reply:
Because its specific and simple enough that it can embedded. I could extract it to the `hack` folder, but I think it will clutter the folder.

This exists because in the CI we will be testing the production deployment so the database is provided by the user. This script mimics the user providing the database.